api/src/main/java/com/alibaba/nacos/api/naming/pojo/healthcheck/impl/Http.java

```diff
@@ -89,7 +89,7 @@ public void setHeaders(final String headers) {
     
     @Override
     public int hashCode() {
-        return Objects.hashCode(path, headers, expectedResponseCode);
+        return Arrays.hashCode(new Object[]{path, headers, expectedResponseCode});
```

Comment:
In `com.google.common.base.Objects` comment, guava recommend 

```
<p><b>Note for Java 7 and later:</b> This method should be treated as deprecated; use {@link
   * java.util.Objects#hash} instead.
```

Reply:
0xa36fD8fF0524f287422337d99bABdf05923f581B